ui/lib/ldap/addon/components/page/role/create-and-edit.ts

```diff
@@ -63,6 +65,9 @@ export default class LdapCreateAndEditRolePageComponent extends Component<Args>
         const action = model.isNew ? 'created' : 'updated';
         yield model.save();
         this.flashMessages.success(`Successfully ${action} the role ${model.name}`);
+        if (action === 'created') {
+          this.store.clearDataset('ldap/role');
```

Comment:
yep, same thing we had to do in kv.

Reply:
It's unfortunate that there's so many pieces so to speak in getting the `lazyPaginatedQuery` to work properly between clearing the data on delete/create and wiring up the page filtering. I hadn't used it in a bit and was originally thinking it would just be a matter of changing `this.store.query` to `this.store.lazyPaginatedQuery` and call it a day!